### PR TITLE
Add node experiences to Technical and Operational page of website. Fixes #21.

### LINKED ISSUE #21: Add nodes' lessons learned in a structured way
**Location**

No specific location, but see below for a discussion of where to place the information.

**Change**

My suggestion is that we define a simple "lessons learned template" where the nodes can briefly summarise their experiences. The headings could look something like this:

* What did we do?
* What went well?
* What could have been done better?
* What did we learn?

I think it is important to keep the text short, so we may want to put restrictions on the number of words under each section. I also think we should avoid adding lessons learned from too many nodes about the same thing.

I'm not sure where this information would be most suitable, but it could be placed under individual topics. The lessons learned regarding outreach and training for example could then be a single file with a structure looking something like this:

* Communication with stakeholders
  * FEGA Poland
    * What did we do?
    * What went well?
    * What could have been done better?
    * What did we learn?
  * FEGA Spain
    * What did we do?
    * What went well?
    * What could have been done better?
    * What did we learn?
* Creating the node's website
  * FEGA Norway
    * What did we do?
    * What went well?
    * What could have been done better?
    * What did we learn?
  * FEGA Sweden
    * What did we do?
    * What went well?
    * What could have been done better?
    * What did we learn?

**Reason**

* New FEGA nodes can avoid some of the misstakes that other nodes did.
* Lessons learned will complement best practices and could potentially serve as a basis for development of new best practices.
* With a defined format, the lessons learned text should be fairly easy for nodes to produce.

### PR DESCRIPTION
**Pull request by malloryfreeberg**
## Description

Node experiences written by representatives of FEGA Finland, Norway, Poland and Sweden were added to the "Technical and Operational" page of the FEGA Onboarding website in the "Learn from current node implementations" section. In this first addition of content, five different experiences are described by 1-3 nodes. Each experience is listed followed by a list of clickable name(s) of a node which expands to show that node's content. Each block of content follows the same template of listing the Author(s) and including 4 subheadings: What did we do?, What went well?, What could have gone better?, and What did we learn?.

## Motivation and Context

Fixes #21. It has been recognised by current and onboarding FEGA nodes that learning from previous nodes' experiences setting up their FEGA node is highly valuable. By sharing node experiences in a structured, static way on the onboarding website, interested nodes can learn from this and hopefully have a more efficient journey joining the FEGA network.

## How has this been tested?

The actual node experience content was simply copied and pasted from a working Google document that all nodes contributed to. To test the expandable content aspect of the website, the site was locally built and served using the method described below.

## What should reviewers focus on?

1. **Review the text for your node's experiences**. Ensure it aligns with what you want to share. Suggest changes if you want to add or change anything. Note that I made a few edits for consistency (e.g. changed all the "Central EGA") but otherwise kept all the content the same.
2. **Test the expandable content**. This feature functions differently in the GitHub markdown versus the Jekyll-rendered website. To test the expandable content experience in the website, you can do the following to test it locally:
    1. In a terminal, `git clone` the FEGA-onboarding repository and navigate to it using `cd FEGA-onboarding`. **NB** If you've cloned the repo previously, simply use `git pull` to get the latest changes.
    2. Switch to the `mf-add-node-experiences` branch locally on your computer using `git checkout mf-add-node-experiences`.
    3. Locally build and serve the website by running `bundle exec jekyll serve`. **NB** You might need to install/update dependencies to use this feature. See more info [here](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll).
    4. Open the local server address (e.g. http://127.0.0.1:4000/FEGA-onboarding/) in a browser, click "docs/", then navigate to the "Node experiences" section e.g. using the left-hand navigation panel. 
    5. Click on the node names to expand/collapse content.
3. **Provide feedback on how the sections looks**. The markdown is a bit hacky, but I tried to get the content to look as nice as possible. If you have suggestions on how to make it look nicer, I will see what I can do (or maybe ask someone to help!)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New content (non-breaking change which adds new content)
- [ ] Modified content (non-breaking change which modifies existing content)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have read the CONTRIBUTING.md guide.
- [x] My changes follow the code style of this project.
- [x] I have updated supporting documentation, if required.
- [x] I have added my details to the CONTRIBUTORS.yaml file.

### CODE REVIEW

File: docs/topics/technical-operational/index.md [hunk 1]
```diff
@@ -83,7 +83,219 @@ Hear more details about node implementations:
...
+<details>
+  <summary><strong>FEGA Poland</strong></summary>
+  Author(s): Krzysztof Kochel<br><br>
+  <h5>What did we do?</h5>
+  
+  After we had established a test environment and validated success stories our Help Desk team started to pass user journeys described in the "Federated EGA node end-to-end demonstrator". Each ambiguity or understatement was written down and sent to the appropriate person in the Central EGA Helpdesk to clarify. When we had confirmed that members of Helpdesk understood each step of each journey, we performed several trials with fresh accounts. This approach allowed us to detect issues which a new user may encounter.<br><br>
+
+  <h5>What went well?</h5>
+  
+  Repeating all the journeys several times with fresh accounts allowed us to detect many issues which would lead to a failure of the final demo.<br><br>
```
Comment (amycurwin) [inline docs/topics/technical-operational/index.md:121]: non-sense text

File: docs/topics/technical-operational/index.md [hunk 2]
```diff
@@ -83,7 +83,219 @@ Hear more details about node implementations:
...
+
+  <h5>What could have gone better?</h5>
+  
+  In some cases we misunderstood explanations. However, those cases were detected and corrected by meeting participants from the Central EGA side.<br><br>
```
Comment (amycurwin) [inline docs/topics/technical-operational/index.md:125]:
```suggestion
  In some cases we misunderstood explanations. Those cases were detected and corrected by meeting participants from the Central EGA side.<br><br>
```

**Review by M-casado — APPROVED** [review]
Reviewed the style and links, but not the content of the nodes' experiences.
I added a few custom changes to the file ``custom.css`` to modify the style of the expandable content, so that it looks more like the following

image

**Comment by amycurwin** [comment]
I really like the changes. I managed to see the changes locally myself and I think this format looks great! Thanks Marcos!

**Review by amycurwin — APPROVED** [review]
Looks great!

**Pull request merged**